common/lib/xmodule/xmodule/util/adaptive_learning.py

```diff
@@ -184,8 +188,12 @@ def create_student(self, student_uid):
         """
         url = self.students_url
         payload = {'uid': student_uid}
-        response = requests.post(url, headers=self.request_headers, data=payload)
-        student = json.loads(response.content)
+        try:
+            student = {}
+            response = requests.post(url, headers=self.request_headers, data=payload)
+            student = json.loads(response.content)
+        except requests.ConnectionError:
+            log.warning("Could not create student identified by student_uid {student_uid}.".format(student_uid=student_uid))
```

Comment:
@macornwell This line is too long; please add some line breaks to make it shorter.